pkg/apps/util/util.go

```diff
@@ -3,599 +3,197 @@ package util
 import (
 	"fmt"
 	"reflect"
-	"sort"
 	"strconv"
-	"strings"
-	"time"
 
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/meta"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/util/diff"
-	"k8s.io/apimachinery/pkg/util/sets"
+	intstrutil "k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
 	scaleclient "k8s.io/client-go/scale"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	autoscalingv1 "k8s.io/kubernetes/pkg/apis/autoscaling/v1"
-	api "k8s.io/kubernetes/pkg/apis/core"
-	kapi "k8s.io/kubernetes/pkg/apis/core"
 	kapiv1 "k8s.io/kubernetes/pkg/apis/core/v1"
-	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
-	kdeplutil "k8s.io/kubernetes/pkg/controller/deployment/util"
 	"k8s.io/kubernetes/pkg/kubectl"
 
-	appsapiv1 "github.com/openshift/api/apps/v1"
+	appv1 "github.com/openshift/api/apps/v1"
 	"github.com/openshift/origin/pkg/api/apihelpers"
-	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
 )
 
-func NewReplicationControllerV1Scaler(client kclientset.Interface) kubectl.Scaler {
-	return kubectl.NewScaler(NewReplicationControllerV1ScaleClient(client))
-}
+const (
```

Comment:
@deads2k looks better now?

I will double check, but we would need to move some of these annotation consts in library-go if we want to move some helpers from this util package over.